parallel-twitter-trending-hashtags/.gitignore

```diff
@@ -0,0 +1 @@
+twitter_wallaroo_app.pyc
```

Comment:
Why not use *.pyc here?

Reply:
Copied from the original. That's the only reason.

Reply:
done